esp-hal/src/soc/esp32/trng.rs

```diff
@@ -0,0 +1,198 @@
+const DR_REG_RTCCNTL_BASE: u32 = 0x3ff48000;
```

Comment:
I know we have previously accepted code like this, but from here on out we should fix the pacs instead of doing this. Chances are many of these fields are already in the pacs.

Reply:
Was a long process of rewriting the whole thing, but also done 😄